src/Features/CSharp/Portable/MakeMethodAsynchronous/CSharpMakeMethodAsynchronousCodeFixProvider.cs

```diff
@@ -36,6 +37,9 @@ protected override string GetMakeAsyncVoidFunctionResource()
         protected override bool IsAsyncSupportingFunctionSyntax(SyntaxNode node)
             => node.IsAsyncSupportingFunctionSyntax();
 
+        protected override bool IsLikelyEntryPointName(string methodName)
+            => string.Equals(methodName, "Main", StringComparison.Ordinal);
```

Comment:
general IDE pattern is to get teh ISyntaxFactsService off of the Document you're starting with, then do syntaxFacts.StringComparer.Equals(...).  This does the right thing for VB and C# and needs no specific delegation like you have here.

Reply:
any reason this is not used?
http://source.roslyn.io/#Microsoft.CodeAnalysis/Compilation/CompilationOptions.cs,44